### ISSUE TITLE
Cannot run tensorflow examples

### ISSUE THREAD

**Issue by Antonio-Sorrentini** [bytedeco/javacpp-presets#754]
F tensorflow/core/framework/op.cc:200] Non-OK-status: RegisterAlreadyLocked(deferred_[i]) status: Already exists: Op with name GRUBlockCell

I've tried many examples unfortunately no one runs.
That error is printed on the console and the program terminates with exit code:  -1073740791 (0xC0000409)

This happens when this line is executed:

Scope scope=Scope.NewRootScope();

Any idea?
Thanks.

**Comment by Antonio-Sorrentini** [comment]
Problem occur when in pom.xml file there is also this:

     <dependency>
         <groupId>org.bytedeco</groupId>
         <artifactId>tensorflow</artifactId>
         <version>1.14.0-1.5.1</version>
         <classifier>windows-x86_64-python</classifier>
     </dependency>

Windows 10 here.
If I remove this it works.

**Comment by saudet** [comment]
Yes, there's currently a limitation on Windows preventing us from using both the C/C++ API and the Python API at the same time.

**Comment by Antonio-Sorrentini** [comment]
Thanks @saudet
